parquet-column/src/main/java/org/apache/parquet/column/values/factory/DefaultV1ValuesWriterFactory.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.parquet.column.values.factory;
+
+import org.apache.parquet.column.ColumnDescriptor;
+import org.apache.parquet.column.Encoding;
+import org.apache.parquet.column.values.ValuesWriter;
+import org.apache.parquet.column.values.plain.BooleanPlainValuesWriter;
+import org.apache.parquet.column.values.plain.FixedLenByteArrayPlainValuesWriter;
+import org.apache.parquet.column.values.plain.PlainValuesWriter;
+
+import static org.apache.parquet.column.Encoding.PLAIN_DICTIONARY;
+
+public class DefaultV1ValuesWriterFactory implements ValuesWriterFactory {
+
+  private ValuesWriterFactoryParams selectionParams;
+
+  @Override
+  public void initialize(ValuesWriterFactoryParams params) {
+    this.selectionParams = params;
+  }
+
+  private Encoding getEncodingForDataPage() {
+    return PLAIN_DICTIONARY;
+  }
+
+  private Encoding getEncodingForDictionaryPage() {
+    return PLAIN_DICTIONARY;
+  }
+
+  @Override
+  public ValuesWriter newValuesWriter(ColumnDescriptor descriptor) {
```

Comment:
I think it was better before, when you could easily see all of the cases that ended up producing a PlainValuesWriter, but this isn't a big deal. If others are fine with this then lets go with it.